langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -124,6 +149,10 @@ public isolated function isReadOnly(anydata v) returns boolean = @java:Method {
 # The details of the conversion are specified by the ToString abstract operation
 # defined in the Ballerina Language Specification, using the direct style.
 #
+# ```ballerina
+# string strValue = 2022.toString();
+# ```
```

Comment:
IMO, its better to declare 2022 as a int variable and then call this method.
Because normally users use this langlib functions for their variables. It is the common use case. wdyt?

Reply:
+1. Applicable to others also.